mta-multi-tenant.yaml

```diff
@@ -47,10 +41,17 @@ modules:
   - name: bookshop-mt-sidecar
 # ------------------------------------------------------------
     type: nodejs
-    path: mtx-sidecar
+    path: mtx/sidecar
     parameters:
       memory: 256M
       disk-quota: 512M
+    build-parameters:
+      builder: custom
+      build-result: gen
+      commands:
+        - npm run build
+      requires:
```

Comment:
Is this build dependency also required in multi-tenant build? I added it for the single-tenant build, to reuse the installed cds-dk from the previous srv-module build.

Reply:
Forget this comment. The sidecar package.json contains just a cds dependency and not cds-dk.